flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Core/A32NX_Core.js

```diff
@@ -63,6 +63,11 @@ class A32NX_Core {
                 module: new A32NX_Speeds(),
                 updateInterval: 500,
             },
+            {
+                name: 'Transition',
+                moduel: new A32NX_Transition(),
```

Comment:
I'm aware that this is a draft, but just thought I should let you know that `module` isn't spelled correctly.
```suggestion
                module: new A32NX_Transition(),
```